### PR TITLE
V2 policy examples

### PR DESCRIPTION
**Pull request by wooglie**
This PR adds three example policies, tests, and documentation explaining each

# Fixed Issuance Policy

Fixed Issuance Policy is an example module of Marmalade which provides a simple method to program a fractional token. Because there are various ways to program fractional tokens, the contract only provides the simplest feature that is required for fractional tokens, and encourages users to extend upon the
contract. 

# Onchain Manifest Policy

The `onchain-manifest-policy-v1` module is a part of Marmalade that provides onchain manifest storage for `marmalade-v2`. It implements the `kip.token-policy-v2` interface and enables the management of token manifests on the blockchain.

# Timed Mint Policy

The Timed Mint Policy is a module of Marmalade that enables the minting of tokens within a specific time window. This policy restricts the minting of tokens to a predefined period and provides control over the minting process.

# Soul Bound Policy
Soul Bound Policy is an example module of Marmalade which provides a simple method to bind a token to an account (soul). In this context, a soul-bound token refers to a token that is uniquely associated with a specific account.

This token can be initiated, minted, and burned but not sold or transferred.

The token can be minted only once, even if the token is burned in the future.

### CODE REVIEW

File: examples/policies/fixed-issuance-policy/fixed-issuance-policy-v1.pact [hunk 1]
```diff
@@ -0,0 +1,123 @@
+(namespace (read-msg 'ns))
+
+(module fixed-issuance-policy-v1 GOVERNANCE
+
+  @doc "Policy for minting with a fixed issuance"
+
+  (defconst ADMIN-KS:string "marmalade-v2.marmalade-contract-admin")
```
Comment (ggobugi27) [inline examples/policies/fixed-issuance-policy/fixed-issuance-policy-v1.pact:7]: We can add 
`marmalade-examples.marmalade-examples-admin` keyset in marmalade.repl (L76~) like the v2 and sale namespaces, and then update ADMIN-KS

File: examples/policies/timed-mint-policy/timed-mint-policy-v1.pact [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  (defcap GOVERNANCE ()
+    (enforce-guard ADMIN-KS))
+
+  (implements kip.token-policy-v2)
+  (use kip.token-policy-v2 [token-info])
+  (use marmalade-v2.policy-manager)
+
+  (defschema timed-mint-schema
+    max-supply:decimal
+    mint-start-time:time
```
Comment (ggobugi27) [inline examples/policies/timed-mint-policy/timed-mint-policy-v1.pact:18]: We're using timestamps in general, so it'd be nice to take in times as integers, in timestamps. Users can use `to-timestamp` function if they'd like to pass in time in time format.

File: examples/policies/timed-mint-policy/timed-mint-policy-v1.pact [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    )
+    (require-capability (MINT-CALL (at "id" token) account amount timed-mint-policy-v1))
+    (let* ( (account-bal:decimal (try 0.0 (at 'balance (marmalade-v2.ledger.details (at 'id token) account))))
+            (total-supply:decimal (try 0.0 (marmalade-v2.ledger.total-supply (at 'id token))))
+            (timed-mint:object{timed-mint-schema} (get-timed-mint token))
+            (max-supply:decimal (at 'max-supply timed-mint))
+            (mint-start-time:time (at 'mint-start-time timed-mint))
+            (mint-end-time:time (at 'mint-end-time timed-mint)) )
+      (enforce (= account-bal 0.0) "Account has already minted")
+      (enforce (>= (at 'block-time (chain-data))  mint-start-time) "Mint has not started yet")
```
Comment (ggobugi27) [inline examples/policies/timed-mint-policy/timed-mint-policy-v1.pact:70]: We can replace `(at 'block-time (chain-data))` to `(curr-time)` function in `util-v1`, which returns the block time in timestamps.

**Review by ggobugi27 — CHANGES_REQUESTED** [review]

File: .github/workflows/test-concrete-policies.yml
[the remark this answers is not in the document]
Comment (wooglie) [reply .github/workflows/test-concrete-policies.yml:1]: You're right, and I think some new general name would be better than splitting the code

**Review by jermaine150 — APPROVED** [review]

**Review by ggobugi27 — APPROVED** [review]

**Pull request merged**